engine/link/cable_club.asm

```diff
@@ -124,21 +124,21 @@ CableClub_DoBattleOrTradeAgain:
 	ld hl, wSerialRandomNumberListBlock
 	ld de, wSerialOtherGameboyRandomNumberListBlock
 	ld bc, $11
-	vc_hook Network17
+	vc_hook Unknown_wireless_ExchangeBytes_1
```

Comment:
Thanks for clarifying what's unknown about this in the PR description; I think that's good info for the repo itself.
```suggestion
	vc_hook Wireless_ExchangeBytes_unknown_Type_5
```

Reply:
I opted for `Wireless_ExchangeBytes_1_unknown_Type5` instead. This denotes it is the first `ExchangeBytes` in the series, even though this has an unknown type 5.